inline_verifier.go

```diff
@@ -613,17 +631,21 @@ func (v *InlineVerifier) compareDecompressedData(source, target map[uint64]map[s
 	return mismatchSet
 }
 
-func (v *InlineVerifier) compareHashesAndData(sourceHashes, targetHashes map[uint64][]byte, sourceData, targetData map[uint64]map[string][]byte) []uint64 {
+func (v *InlineVerifier) compareHashesAndData(sourceHashes, targetHashes map[uint64][]byte, sourceData, targetData map[uint64]map[string][]byte) []InlineVerifierMismatches {
 	mismatches := v.compareHashes(sourceHashes, targetHashes)
 	compressedMismatch := v.compareDecompressedData(sourceData, targetData)
 	for paginationKey, _ := range compressedMismatch {
-		mismatches[paginationKey] = struct{}{}
+		mismatches[paginationKey] = InlineVerifierMismatches{
+			Pk:             paginationKey,
+			SourceChecksum: "compressed-data-mismatch", // TODO: compute the hash of the compressed data and put it here
+			TargetChecksum: "compressed-data-mismatch",
```

Comment:
Just wondering why ship this without the computed compressed data hashes.  Does it just seem much more likely to hit `compareHashes` mismatches?

Reply:
It's slightly complex to calculate this for now. Where we are observing this is also not associated with this code path. So for now we'll just ignore it.